src/ar.erl

```diff
@@ -123,6 +127,8 @@ main("") ->
 			{"new_mining_key", "Generate a new keyfile, apply it as the reward address"},
 			{"load_mining_key (file)", "Load the address that mining rewards should be credited to from file."},
 			{"content_policy (file)", "Load a content policy file for the node."},
+			{"transaction_blacklist (file)", "A .txt file containing blacklisted transactions. "
+											 "Every line has to be a Base64 encoded transaction ID."},
```

Comment:
Here is my suggested improvement of the description:

A .txt file containing blacklisted transactions. One Base64 encoded transaction ID per line.

Reply:
✅